### ISSUE TITLE
Checklists: Allow sorting and removal of done tasks

### ISSUE THREAD

**Issue by ix5** [federicoiosue/Omni-Notes#419]
It would be nice to have the option to sort by status(done or open) for checklists, as well as to have an option to remove done tasks.
This would really alleviate some of my pains with long lists!

**Comment by federicoiosue** [comment]
I usually do this by selecting to transform checklist back to simple note and selecting the desired options from the dialog.

![screenshot_2018-01-05-16-21-17](https://user-images.githubusercontent.com/487104/34615280-a643ad4e-f234-11e7-9e0a-b805d3f1d6cc.png)

For what concerns items' moving up and down it depends from the global setting but I've no plans about implementing a specific behavior for every single note.

**Comment by ix5** [comment]
Thank you federico, I did not realize this could be accomplished already.

For others: Go to `Settings` - `Behavious` - `Checklists` and check "Move to end [of unchecked items]" and your todo-lists will be sorted automatically.

It is just really unintuitive, we ought to have a menu entry in checklist mode that says "Remove checked items" instead, IMHO.

**Comment by federicoiosue** [comment]
I agree, eventually adding a new entry into the overflow menu  would help
